src/buckets/components/createBucketForm/UpdateBucketOverlay.tsx

```diff
@@ -70,10 +79,16 @@ const UpdateBucketOverlay: FunctionComponent<Props> = ({
         handleClose()
         return
       }
+
       setBucketDraft(resp.data as OwnBucket)
 
       setSchemaType(resp.data.schemaType)
 
+      if ('explicit' === resp.data.schemaType) {
```

Comment:
oh, you don't see a lot of yoda conditionals these days. i don't think we do that anywhere and i think with our lint rules, we're safe from accidentally doing `resp.data.schemaType = 'explicit'`

Reply:
are you saying you don't like this style?  I prefer this style so put it in when I write new code

Reply:
(do you want me to reverse this, or don't bother?) thanks @hoorayimhelping

Reply:
i'm generally leery of introducing one off patterns, even if they're better because of situations like this:

![](https://imgs.xkcd.com/comics/standards.png)